### ISSUE TITLE
[Bug]: GetOpenSubsonicExtensions returning no extensions

### ISSUE THREAD

**Issue by dweymouth** [navidrome/navidrome#2785]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.50.2

### Current Behavior

Calling the `getOpenSubsonicExtensions` endpoint returns no extensions

```
GET https://demo.navidrome.org/rest/getOpenSubsonicExtensions?u=demo&p=demo&v=1.15.0&c=supersonic

<subsonic-response status="ok" version="1.16.1" type="navidrome" serverVersion="0.50.2 (823bef54)" openSubsonic="true">
<openSubsonicExtensions/>
</subsonic-response>
```

### Expected Behavior

It should return the extensions supported by Navidrome (transcodeOffset, etc)

### Steps To Reproduce

Do a GET on the URL in the "Current Behavior" section (or just visit it in a web browser)

### Environment

```markdown
- OS:
- Browser:
- Client:
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by kgarner7** [comment]
These are already in master. see https://github.com/navidrome/navidrome/commit/7766ee069cb50f298144ffd46ea0fd93e3540b79, https://github.com/navidrome/navidrome/commit/812dc2090f20ac4f8ac271b6ed95be5889d1a3ca, and https://github.com/navidrome/navidrome/commit/814161d78db0dc07c16fa6e80f6ff1710866e31b

**Comment by dweymouth** [comment]
Ah it just hasn't landed in a release yet? I thought 0.50.2 had support for the first of the extensions
